### PR TITLE
[#57592] 14.5 Documentation: Rename actions in work package dropdown …

### PR DESCRIPTION
**Pull request by MayaBerd**
…menu  https://community.openproject.org/work_packages/57592

[#57592] 14.5 Documentation: Rename actions in work package dropdown menu

https://community.openproject.org/work_packages/57592

# Ticket

# What are you trying to accomplish?

## Screenshots

# What approach did you choose and why?

# Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation in Lookbook (patterns, previews, etc)
- [ ] Tested major browsers (Chrome, Firefox, Edge, ...)

### CODE REVIEW

**Comment by MayaBerd** [comment]
@as-op could you please take a look at Links and URL changes triggered by the change in terms? Thank you!

File: docs/user-guide/work-packages/duplicate-move-delete/README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 
-If you right-click in a work package table, editing options will be displayed. Here, you can change project, copy the work package, or copy its URL to the clipboard, delete it or move it to another project.
+> [!TIP] 
+>
+> In OpenProject 14.5 the term *Copy a work package* was replaced by *Duplicate a work package*.
+>
+> *Change project* was replaced by *Move to another project*.
 
-![change copy delete work package](openproject_user_guide_copy_move_delete_wptableview.png)
+If you right-click in a work package table, editing options will be displayed. Here, you can move a work package to another project, duplicate a work package, or copy its URL to the clipboard, delete it or duplicate it to another project.
```
Comment (psatyal) [inline docs/user-guide/work-packages/duplicate-move-delete/README.md]: Here, I would rephrase the last bit: "... duplicate it in another project" (vs "to").

**Review by psatyal — CHANGES_REQUESTED** [review]
Hi @MayaBerd, just one small change but looks good otherwise.

Cheers
Parimal

File: docs/user-guide/work-packages/duplicate-move-delete/README.md (hunk 1 above)
Comment (MayaBerd) [inline docs/user-guide/work-packages/duplicate-move-delete/README.md]:
```suggestion
If you right-click in a work package table, editing options will be displayed. Here, you can move a work package to another project, duplicate a work package, or copy its URL to the clipboard, delete it or duplicate it in another project.
```
